cmd/istioctl/main.go

```diff
@@ -28,8 +32,19 @@ import (
 
 	"istio.io/manager/cmd"
 	"istio.io/manager/model"
+
+	"k8s.io/client-go/pkg/util/yaml"
 )
 
+// Each entry in the multi-doc YAML file used by `istioctl create -f` MUST have this format
+type inputDoc struct {
+	Type string
+	Name string
+	// Spec is usually, but not always, a istio.proxy.v1alpha.config.RouteRule
+	Spec       interface{}
```

Comment:
I would also remove the comment above spec. Because user can provide route-rule or destination-policy via the CLI (and other types in future).